views/FanView/FanView.tsx

```diff
@@ -24,26 +33,29 @@ export const FanView: FC<FanViewProps> = ({ route }) => {
   ) as Fan;
 
   const [temp, onTempChange] = useInputValue(fan.values.V_TEMP);
-  const [direction, onDirectionChange] = useInputValue(fan.values.V_DIRECTION);
   const [percentage, onPercentageChange] = useInputValue(
     fan.values.V_PERCENTAGE,
   );
+  const [direction, onDirectionChange] = useInputValue(fan.values.V_DIRECTION);
 
   const handleChangeFanParams = () => {
+    const finalTempValue = parseFloat(temp).toString();
+    const finalDirectionValue = parseFloat(direction).toString();
+    const finalPercentageValue = parseFloat(percentage).toString();
```

Comment:
Why?

Reply:
To avoid situation like when in input is something like `54.`